src/pkcs11/framework-pkcs15.c

```diff
@@ -3835,6 +3897,11 @@ pkcs15_prkey_get_attribute(struct sc_pkcs11_session *session,
 	case CKA_MODULUS_BITS:
 		check_attribute_buffer(attr, sizeof(CK_ULONG));
 		switch (prkey->prv_p15obj->type) {
+			case SC_PKCS15_TYPE_PRKEY_EDDSA:
+			case SC_PKCS15_TYPE_PRKEY_XEDDSA:
+				/* TODO where to get field length ? */
+				*(CK_ULONG *) attr->pValue = 255;
```

Comment:
I don't have an answer to this. Do you think we need to solve this before merging?

Reply:
At this moment, we do not support anything else than 255 bits curves. Also other places have just the constant 255 for now. The EdDSA keys are just bytes represented in  `sc_pkcs15_pubkey_eddsa.pubkey` and `sc_pkcs15_prkey_eddsa.pubkey` so we should be able to derive the length from there, but the issue is that PKCS #11 3.0 specifies the length as 255, while generally we would get 256 from simply calculating bytes. Or we can store it in separate field of the above structures. I can add it if you would like. It might clarify also some of the other places.